plugins/callback/opentelemetry.py

```diff
@@ -276,6 +271,22 @@ def set_span_attribute(self, span, attributeName, attributeValue):
             if attributeValue is not None:
                 span.set_attribute(attributeName, attributeValue)
 
+    def get_error_message(self, res):
+        message = ''
+        if res.get('exception') is not None:
```

Comment:
It probably makes more sense to test whether this is actually a string:
```suggestion
        if isinstance(res.get('exception'), str):
```